src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -56,6 +56,9 @@ public ActionDefinitionData Load(IExecutionContext executionContext, string mani
             var templateContext = CreateContext(executionContext);
             ActionDefinitionData actionDefinition = new ActionDefinitionData();
 
+            // Set ActionManifestSchema so that we can use it for post processing in Composite Actions
+            executionContext.ActionManifestSchema = templateContext.Schema;
```

Comment:
instead of adding to execution context, lets expose from IActionManifestManager

Consumer can access like:

```csharp
var actionManifestManager = HostContext.GetService<IActionCommandManager>().ActionManifestSchema;
actionManifestManager.ActionManifestSchema;
```